### ISSUE TITLE
New Concept: Delay-Dopper Imaging

### ISSUE THREAD

**Issue by mkelley** [astrothesaurus/UAT#272]
**Name the new concept**
Delay-Doppler imaging

**Describe the concept**
A specific technique in "Radar astronomy," delay-Doppler imaging can determine size, shape, rotation, and radar albedo for a spatially resolved object.

**Describe where the concept fits within the existing hierarchy**
Observational astronomy : Astronomical methods : Radar astronomy : Delay-Doppler imaging

**Provide 1-3 supporting articles**
2010Icar..207..499H
2004come.book..265H
1992Sci...258..640H

**Comment by BartlettAstro** [comment]
@mkelley can you provide a definition from an authoritative source?  We are trying to ensure new entries have formal definitions.

**Comment by sjarmak** [comment]
I agree with the suggested hierarchy. Michael Kelley is a fairly authoritative source on this, but if part of the definition needs to have a reference or more formal language around the physical process involved with delay-doppler imaging we could use, 

"A specific technique in "Radar astronomy," delay-Doppler imaging can determine size, shape, rotation, and radar albedo for a spatially resolved object. To spatially resolve the object, the target is illuminated by a series of transmitted pulses. The return for each pulse is a superposition of reflections from various locations on the target, with each location affecting the pulse by introducing both a delay and a Doppler shift. The returns are processed to produce an image of the target." the latter part comes from Donald L. Snyder 1987 Semiannual progress report.

**Comment by mkelley** [comment]
To my knowledge this is a good definition.  I can provide or contact a few people who actually work with radar data, if you want a more detailed review.
